src/Workspaces/CSharp/Portable/LanguageServices/CSharpParseOptionsService.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.LanguageServices;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    [ExportLanguageService(typeof(IParseOptionsService), LanguageNames.CSharp), Shared]
+    internal class CSharpParseOptionsService : IParseOptionsService
```

Comment:
The VB scenario cannot be lit up yet, because I couldn't find a DTE API to update the `LanguageVersion`. Any tips how to handle that?